### PR TITLE
Add ANZT6S tournament page

### PR DESCRIPTION
**Pull request by JWWolstenholme**

### CODE REVIEW

File: wiki/Tournaments/ANZT/2019_Summer/en.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+[spreadsheet]: https://docs.google.com/spreadsheets/d/1gnV5_7e-57MXtpoY57lgyPKyF3_IvjDXsGoLVgrnqk8
+
+# Aus/NZ osu!standard Tournament 6 - Summer
+
+![ANZT 6S Logo](img/logo.png)
+
+The **Aus/NZ osu!standard Tournament 6 - Summer** (***ANZT 6S***) was a single elimination one-on-one osu!standard tournament with a Swiss format group stage hosted by ![][flag_NZ] [Diony](https://osu.ppy.sh/users/4294475) and ![][flag_AU] [EmeraldStar82](https://osu.ppy.sh/users/3772135). It was the 7th installment of Aus/NZ tourneys.
```
Comment (fajar13k) [inline wiki/Tournaments/ANZT/2019_Summer/en.md]: Is ANZT 6S really the 7th installment?

File: wiki/Tournaments/ANZT/2019_Summer/en.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 4 months of osu!supporter tag, unique profile badge |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 2 months of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 month of osu!supporter tag |
+
+## Organisation
+
+| Position | Member(s) |
+| :-- | :-- |
+| Management | ![][flag_NZ] [Diony](https://osu.ppy.sh/users/4294475), ![][flag_AU] [EmeraldStar82](https://osu.ppy.sh/users/3772135) |
+| Scheduling | ![][flag_AU] [EmeraldStar82](https://osu.ppy.sh/users/3772135),  |
```
Comment (fajar13k) [inline wiki/Tournaments/ANZT/2019_Summer/en.md]:
```suggestion
| Scheduling | ![][flag_AU] [EmeraldStar82](https://osu.ppy.sh/users/3772135) |
```

File: wiki/Tournaments/ANZT/2019_Summer/en.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+| Commentators | ![][flag_AU] [abolaspany](https://osu.ppy.sh/users/4261853), ![][flag_AU] [awesometa](https://osu.ppy.sh/users/7114425), ![][flag_NZ] [Monster Bait](https://osu.ppy.sh/u/6890546), ![][flag_AU] [Monk Gyatso](https://osu.ppy.sh/u/4012086), ![][flag_AU] [Arbusion](https://osu.ppy.sh/users/11469447), ![][flag_AU] [HolyJesus](https://osu.ppy.sh/users/7785655), ![][flag_AU] [Aus](https://osu.ppy.sh/users/6541404), ![][flag_AU] [Takagaki](https://osu.ppy.sh/users/3698691), ![][flag_AU] [pewdekz](https://osu.ppy.sh/users/5754859) |
+| Streamers | ![][flag_AU] [pewdekz](https://osu.ppy.sh/users/5754859), ![][flag_VN] [steve_04_](https://osu.ppy.sh/users/10852911), ![][flag_AU] [TheOmyNomy](https://osu.ppy.sh/users/4241054) |
```
Comment (fajar13k) [inline wiki/Tournaments/ANZT/2019_Summer/en.md]:
```suggestion
| Streamers | ![][flag_AU] [pewdekz](https://osu.ppy.sh/users/5754859), ![][flag_VN] [steve\_04\_](https://osu.ppy.sh/users/10852911), ![][flag_AU] [TheOmyNomy](https://osu.ppy.sh/users/4241054) |
```

File: wiki/Tournaments/ANZT/2019_Summer/en.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+| Referees | ![][flag_AU] [Illinon](https://osu.ppy.sh/users/8306102), ![][flag_AU] [Soulskyeflare](https://osu.ppy.sh/users/8264319), ![][flag_AU] [awesometa](https://osu.ppy.sh/users/7114425), ![][flag_PH] [ggnakacarlz](https://osu.ppy.sh/users/4075421), ![][flag_NZ] [phophicles](https://osu.ppy.sh/users/9373724), ![][flag_VN] [steve_04_](https://osu.ppy.sh/users/10852911), ![][flag_NZ] [NutAndGo](https://osu.ppy.sh/users/7740118), ![][flag_KR] [silverlightcs](https://osu.ppy.sh/users/9862564), ![][flag_NZ] [Div](https://osu.ppy.sh/users/3751116), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811) |
```
Comment (fajar13k) [inline wiki/Tournaments/ANZT/2019_Summer/en.md]:
```suggestion
| Referees | ![][flag_AU] [Illinon](https://osu.ppy.sh/users/8306102), ![][flag_AU] [Soulskyeflare](https://osu.ppy.sh/users/8264319), ![][flag_AU] [awesometa](https://osu.ppy.sh/users/7114425), ![][flag_PH] [ggnakacarlz](https://osu.ppy.sh/users/4075421), ![][flag_NZ] [phophicles](https://osu.ppy.sh/users/9373724), ![][flag_VN] [steve\_04\_](https://osu.ppy.sh/users/10852911), ![][flag_NZ] [NutAndGo](https://osu.ppy.sh/users/7740118), ![][flag_KR] [silverlightcs](https://osu.ppy.sh/users/9862564), ![][flag_NZ] [Div](https://osu.ppy.sh/users/3751116), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811) |
```

File: wiki/Tournaments/ANZT/2019_Summer/en.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+| Graphics Design | ![][flag_AU] [abolaspany](https://osu.ppy.sh/users/4261853) |
+
+## Links
+
+- [Discussion thread](/community/forums/topics/839284)
+- [Livestream](https://www.twitch.tv/osuanzt)
+
+## Mappools
+
+Find all mappools on the [main spreadsheet][spreadsheet].
```
Comment (fajar13k) [inline wiki/Tournaments/ANZT/2019_Summer/en.md]:
```suggestion
Find all mappools on the [main spreadsheet](spreadsheet).
```
Also, where's the link?

File: wiki/Tournaments/ANZT/2019_Summer/en.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+| Streamers | ![][flag_AU] [pewdekz](https://osu.ppy.sh/users/5754859), ![][flag_VN] [steve_04_](https://osu.ppy.sh/users/10852911), ![][flag_AU] [TheOmyNomy](https://osu.ppy.sh/users/4241054) |
+| Referees | ![][flag_AU] [Illinon](https://osu.ppy.sh/users/8306102), ![][flag_AU] [Soulskyeflare](https://osu.ppy.sh/users/8264319), ![][flag_AU] [awesometa](https://osu.ppy.sh/users/7114425), ![][flag_PH] [ggnakacarlz](https://osu.ppy.sh/users/4075421), ![][flag_NZ] [phophicles](https://osu.ppy.sh/users/9373724), ![][flag_VN] [steve_04_](https://osu.ppy.sh/users/10852911), ![][flag_NZ] [NutAndGo](https://osu.ppy.sh/users/7740118), ![][flag_KR] [silverlightcs](https://osu.ppy.sh/users/9862564), ![][flag_NZ] [Div](https://osu.ppy.sh/users/3751116), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811) |
+| Mappool Selectors | ![][flag_JP] [Delis](https://osu.ppy.sh/users/1603923), ![][flag_JP] [Angel Arrow](https://osu.ppy.sh/users/1970239), ![][flag_MV] [HypebeastHijabi](https://osu.ppy.sh/users/1786945) |
+| Graphics Design | ![][flag_AU] [abolaspany](https://osu.ppy.sh/users/4261853) |
+
+## Links
+
+- [Discussion thread](/community/forums/topics/839284)
```
Comment (fajar13k) [inline wiki/Tournaments/ANZT/2019_Summer/en.md]:
```suggestion
- [Discussion thread](https://osu.ppy.sh/community/forums/topics/839284)
```

File: wiki/Tournaments/ANZT/2019_Summer/en.md (hunk 1 above)
Comment (JWWolstenholme) [reply wiki/Tournaments/ANZT/2019_Summer/en.md]: Confusingly, yes. There was ANZT 1 through 4, then ANZT 5S & 5W, and now 6S

File: wiki/Tournaments/ANZT/2019_Summer/en.md (hunk 6 above)
Comment (JWWolstenholme) [reply wiki/Tournaments/ANZT/2019_Summer/en.md]: I think the original is correct here looking at "Other osu! web pages" under Links on the styling criteria.

File: wiki/Tournaments/ANZT/2019_Summer/en.md (hunk 5 above)
Comment (JWWolstenholme) [reply wiki/Tournaments/ANZT/2019_Summer/en.md]: This is a reference style link mentioned under 'Links' on the styling criteria. Flag icons for usernames are the same: `![][flag_AU]`, but I don't know what the exclamation mark does.

It appears they don't function without square brackets hence 4df9faec09e46479ab4310204872bad342b035a9.

File: wiki/Tournaments/ANZT/2019_Summer/en.md (hunk 6 above)
Comment (cl8n) [reply wiki/Tournaments/ANZT/2019_Summer/en.md]: we'll update that soon, but this is now what we're enforcing. everything besides /wiki should have the hostname

File: wiki/Tournaments/ANZT/2019_Summer/en.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+| Referees | ![][flag_AU] [Illinon](https://osu.ppy.sh/users/8306102), ![][flag_AU] [Soulskyeflare](https://osu.ppy.sh/users/8264319), ![][flag_AU] [awesometa](https://osu.ppy.sh/users/7114425), ![][flag_PH] [ggnakacarlz](https://osu.ppy.sh/users/4075421), ![][flag_NZ] [phophicles](https://osu.ppy.sh/users/9373724), ![][flag_VN] [steve\_04\_](https://osu.ppy.sh/users/10852911), ![][flag_NZ] [NutAndGo](https://osu.ppy.sh/users/7740118), ![][flag_KR] [silverlightcs](https://osu.ppy.sh/users/9862564), ![][flag_NZ] [Div](https://osu.ppy.sh/users/3751116), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811) |
+| Mappool Selectors | ![][flag_JP] [Delis](https://osu.ppy.sh/users/1603923), ![][flag_JP] [Angel Arrow](https://osu.ppy.sh/users/1970239), ![][flag_MV] [HypebeastHijabi](https://osu.ppy.sh/users/1786945) |
+| Graphics Design | ![][flag_AU] [abolaspany](https://osu.ppy.sh/users/4261853) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/839284)
+- [Livestream](https://www.twitch.tv/osuanzt)
```
Comment (Joehuu) [inline wiki/Tournaments/ANZT/2019_Summer/en.md]:
```suggestion
- [Livestream](https://www.twitch.tv/osuanzt)
- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1gnV5_7e-57MXtpoY57lgyPKyF3_IvjDXsGoLVgrnqk8)**
```

**Comment by JWWolstenholme** [comment]
@Joehuu Are reference style links like that spreadsheet link not recommended anymore or something?

**Comment by Joehuu** [comment]
Usually spreadsheets in tournament articles are only linked once, being the `Statistics sheet` and referencing the sheet in other parts something along the lines `statistics sheet linked above` (with no actual link).

Also, reference style links only apply to images not links. I think this is because we should not duplicate linked stuff that is already provided in the same article.

**Pull request merged**